pkg/sfu/sfu.go

```diff
@@ -151,6 +155,14 @@ func init() {
 
 // NewSFU creates a new sfu instance
 func NewSFU(c Config) *SFU {
+	// Init loggers
+	if c.Logger == nil {
+		println("Please pass logr instance implementation in SFU Config. You can use https://github.com/pion/ion-sfu/blob/5e6f7a3f58f0411d65ef4b31e8b6d58b6afd16da/pkg/logger/zerologr.go")
+		os.Exit(1)
+	}
+	defaultLogger = c.Logger
+	debugLogger = c.Logger.V(1)
```

Comment:
We should avoid globals like this.  Inject c.Logger into whatever needs it (e.g. NewBufferFactory below should take a logger argument).

Reply:
Done

Reply:
Added.